app/customer.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any
+from app.shop import Shop
+from app.car import Car
+
+
+class Customer:
+    def __init__(self,
+                 client_name: str,
+                 client_products: dict,
+                 location: list,
+                 money: float,
+                 car: dict) -> None:
+        self.client_name = client_name
+        self.client_products = client_products
+        self.location = location
+        self.money = money
+        self.car = Car(car["brand"], car["fuel_consumption"])
+
+    def total_cost(self, shop: Shop, fuel_price: float) -> float:
+        distance = self.car.calculate_distance(self.location, shop.location)
+        fuel_cost = self.car.fuel_cost(distance * 2, fuel_price)
+        product_cost = shop.products_cost(self.client_products)

```

Comment:
The `products_cost` method in the `Shop` class returns -1 if a product is not available. You should handle this case in the `total_cost` method to avoid incorrect calculations.